src/ensembl/src/header/launchbar/Launchbar.tsx

```diff
@@ -74,28 +74,22 @@ const Launchbar = () => {
               enabled={true}
             />
           </div>
-          <div className={styles.category}>
-            <LaunchbarButton
-              app="custom-download"
-              description="Downloads"
-              icon={CustomDownloadIcon}
-              enabled={
-                isEnvironment([
-                  Environment.DEVELOPMENT,
-                  Environment.INTERNAL
-                ]) && committedSpecies.length > 0
-              }
-            />
-          </div>
+          {isEnvironment([Environment.DEVELOPMENT, Environment.INTERNAL]) && (
```

Comment:
Do we need to completely hide the custom download launchbar icon on production?

Reply:
That was the idea, yes. We can verify with Andrea and Steve.